definitions/variable/tag_electricity_storage_types.yaml

```diff
@@ -0,0 +1,32 @@
+# This list of inputs will replace any occurrence of `{Energy Storage System Type}`
+# in a variable-definition yaml file
+
+# This list is intended for use as input energy storage system to the power sector
+
+- Electricity Storage Type:
+
+  - Energy Storage System:
+      description: Mainly to refer to the batteries
+  - Energy Storage System|Lithium-Ion:
+      description: Lithium-Ion batteries
+  - Energy Storage System|Redox Flow:
+      description: Redox Flow batteries
+  - Energy Storage System|Compressed Air:
+      description: Compressed Air batteries
+
+  - Hydro:
+      description: hydropower
+  - Hydro|Pumped Storage:
+      description: hydropower systems with short-term storage capacity
+        (including pumping turbines)
+      notes: This hydropower system has two water reservoirs at different elevations.
+        The definition follows the convention established
+        by the ENTSO-E transparency platform (https://transparency.entsoe.eu).
+  - Hydro|Reservoir:
+      description: hydropower systems with seasonal storage capacity
+        (possibly including pumping turbines)
+      notes: This definition follows the convention established
+        by the ENTSO-E transparency platform (https://transparency.entsoe.eu).
+
+  - Solar|CSP:
```

Comment:
Why is Concentrated Solar Power seen as a battery?

Reply:
I think it is not totally defined since it should refer to the 'Solar|CSP|With Thermal Energy Storage'.
Do you also agree to define the following sub-categories for solar CSP: 
- `Solar|CSP|With Thermal Energy Storage`
- `Solar|CSP|Without Thermal Energy Storage`

Reply:
But I still don’t think that CSP with (heat) storage should be listed as a storage technology. CSP with storage can smooth the generation profile, but you cannot inject electricity from the grid into that storage. It should be seen as just a generation technology.

Reply:
I agree with @danielhuppmann ; Electrcity Storage Types is meant to list the different kinds of electricity storages, ie where you inject/discharge electricity; except for hydro of course, but in the models we treat hydro as if it was storing electricity (as we virtually inject electricity when consuming electricity to take water upstream); Here no electricity is injected, so I would remove this variable (or we should add all heat storages....)

Reply:
> I think it is not defined since it should refer to the 'Solar|CSP|With Thermal Energy Storage'. Do you also agree to define the following sub-categories for solar CSP?:
> 
> * `Solar|CSP|With Thermal Energy Storage`
> * `Solar|CSP|Without Thermal Energy Storage`

These tags 'w/ Thermal Energy Storage', 'w/o Thermal Energy Storage' may be usefull but not in a tag named 'electricity storage types'. But I do not know wether models dealing with thermal storages would need that.... So I do not have an opinion except that it should not be in the current file.

Reply:
I've just moved `Solar|CSP` to tag_electricity_input_types.yaml.
Then, the file `tag_electricity_storage_types.yaml` comprises the following list:

```
- Electricity Storage Type:

  - Energy Storage System:
      description: Mainly to refer to the batteries
  - Energy Storage System|Lithium-Ion:
      description: Lithium-Ion batteries
  - Energy Storage System|Redox Flow:
      description: Redox Flow batteries
  - Energy Storage System|Compressed Air:
      description: Compressed Air batteries

  - Hydro:
      description: hydropower
  - Hydro|Pumped Storage:
      description: hydropower systems with short-term storage capacity
        (including pumping turbines)
      notes: This hydropower system has two water reservoirs at different elevations.
        The definition follows the convention established
        by the ENTSO-E transparency platform (https://transparency.entsoe.eu).
  - Hydro|Reservoir:
      description: hydropower systems with seasonal storage capacity
        (possibly including pumping turbines)
      notes: This definition follows the convention established
        by the ENTSO-E transparency platform (https://transparency.entsoe.eu).
```


@sandrinecharousset, @danielhuppmann, now, is the PR ready to be merged?